CMakeLists.txt

```diff
@@ -3,7 +3,8 @@ project(libkeyfinder)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-option(BUILD_SHARED_LIBS "Build dynamic library" OFF)
+option(BUILD_SHARED_LIBS "Build dynamic library" ON)
```

Comment:
I agree with this change but the commit message is the opposite.

Reply:
Fixed.